website/projects/awssync.py

```diff
@@ -178,3 +233,62 @@ def attach_scp_policy(self, policy_id, target_id):
             self.logger.error("Something went wrong attaching an SCP policy to a target.")
             self.logger.debug(f"{error}")
             self.logger.debug(f"{error.response}")
+
+    # TODO: check if this function is really needed
+
+    def check_for_double_member_email(self, aws_list: list[SyncData], sync_list: list[SyncData]):
+        """
+        Check if no users are assigned to multiple projects.
+
+        Input will be a list of SyncData objects.
+        """
+        sync_emails = [x.project_email for x in sync_list]
+        aws_emails = [x.project_email for x in aws_list]
+        duplicate = None
+
+        for x in sync_emails:
+            if x in aws_emails:
+                duplicate = x
+                break
```

Comment:
Why not check for all email? Taking intersection and checking if empty is oneliner, and otherwise you can log all double emails. Imagine you run the pipeline and this edge case gets triggered, then (manually) fix it, run pipeline second time, and again this edge case gets triggered for a different email, as opposed to fixing all in one go.